src/hive_mind_bridge/src/main.cpp

```diff
@@ -12,7 +12,7 @@ int main(int argc, char** argv) {
     ros::init(argc, argv, "hive_mind_bridge");
     ros::NodeHandle nodeHandle;
 
-    std::string robotName = ros::param::param("~ROBOT_NAME", std::string("pioneer_0"));
+    std::string robotName = ros::param::param("~", std::string("agent_1"));
     ros::Publisher moveByPublisher = nodeHandle.advertise<swarmus_ros_navigation::MoveByMessage>(
         robotName + "/navigation/moveBy", 1000);
```

Comment:
With the new namespace feature you should actually remove the robotName string and the ROBOT_NAME param. Because with this implementation a `/agent_1/agent_1/navigation/moveBy` is published instead of `/agent_1/navigation/moveBy` (there is an extra prefix).